grails-app/assets/javascripts/spApp/service/toolAddFacetService.js

```diff
@@ -25,22 +25,22 @@
                             }
                         },
                         {
-                            "description": "Species options.",
+                            "description": $i18n(411),
```

Comment:
Shall we use a default value, like: $i18n(411, "Species options."). It would give developers a direct hint : it is "Species options."